src/TagHeaderFormatter/TagHeaderFormatterInterface.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCache\TagHeaderFormatter;
+
+/**
+ * Interface for tag header formatting.
+ *
+ * @author Yanick Witschi <yanick.witschi@terminal42.ch>
+ */
+interface TagHeaderFormatterInterface
```

Comment:
this project follows the convention of naming interfaces without the `Interface` postfix. please rename this to TagHeaderFormatter.